src/css/components/_tracks-button.scss

```diff
@@ -0,0 +1,31 @@
+// North America uses 'CC' icon
+.video-js:lang(en) .vjs-tracks-button .vjs-icon-placeholder,
+.video-js:lang(fr-CA) .vjs-tracks-button .vjs-icon-placeholder {
+  @extend .vjs-icon-captions;
+}
+
+// ROW uses 'subtitles'
+// Double selector because @extend puts these rules above the captions icon
+.video-js .vjs-tracks-button .vjs-icon-placeholder,
+.video-js.video-js:lang(en-GB) .vjs-tracks-button .vjs-icon-placeholder,
+.video-js.video-js:lang(en-IE) .vjs-tracks-button .vjs-icon-placeholder,
+.video-js.video-js:lang(en-AU) .vjs-tracks-button .vjs-icon-placeholder,
+.video-js.video-js:lang(en-NZ) .vjs-tracks-button .vjs-icon-placeholder {
+  @extend .vjs-icon-subtitles;
+}
+
+// Todo
+.video-js .vjs-tracks-button .vjs-captions-menu-item::after {
```

Comment:
so, `captions` are always indicated as `[cc]` and subs are only indicated as such in `en` and `fr-CA` locales?